app/controllers/account_controller.rb

```diff
@@ -172,7 +172,8 @@ def activate
   end
 
   def handle_expired_token(token)
-    send_activation_email! Token::Invitation.create!(user: token.user)
+    new_token = Token::Invitation.create!(user: token.user)
+    UserMailer.user_signed_up(new_token).deliver_later
```

Comment:
I inlined it because adding it in a method has no advantages. It's not being reused and does not save any lines. I'll revert this change if you prefer.

Reply:
That's alright then.